### PR TITLE
single top s_channel cards for 2017 production

### PR DESCRIPTION
**Pull request by ckmackay**
Please can you check these cards: single top s-channel leptonic for 2017 production?

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_sch_lept_4f_ckm_NLO/st_sch_lept_4f_ckm_NLO_madspin_card.dat [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+#*                                                          *
+#************************************************************
+#Some options (uncomment to apply)
+#
+#directory for gridpack mode
+set ms_dir ./madspingrid
+
+#initialization parameters
+ set Nevents_for_max_weigth 250 # number of events for the estimate of the max. weight
+# set BW_cut 15                # cut on how far the particle can be off-shell
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_sch_lept_4f_ckm_NLO/st_sch_lept_4f_ckm_NLO_madspin_card.dat]: please remove this commented input

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_sch_lept_4f_ckm_NLO/st_sch_lept_4f_ckm_NLO_proc_card.dat [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import model loop_sm-ckm
+#switch to non-diagonal ckm matrix for speed
```
Comment (efeyazgan) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/st_sch_lept_4f_ckm_NLO/st_sch_lept_4f_ckm_NLO_proc_card.dat]: Please remove these 2 commented lines as well.

**Comment by ckmackay** [comment]
Removed the commented lines

**Comment by efeyazgan** [comment]
Should the ren. and fact. scales really be fixed?

**Comment by ckmackay** [comment]
These are the previous cards updated, I'm not sure, does anyone else know?

**Comment by efeyazgan** [comment]
Was it like that, e.g. in TOP-RunIISummer15wmLHEGS-00211 ? May be you can check with Alexander and/or Reza.

**Comment by ckmackay** [comment]
yes, they are the same cards, I used the ./copyOldCardsFor2017.sh script to update the previously used cards

**Comment by ckmackay** [comment]
Checking with Benedikt who made the original cards

**Comment by efeyazgan** [comment]
I see the opposite if I look at genproductions-pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_tch_4f_ckm_NLO from
https://github.com/cms-sw/genproductions/releases/tag/pre2017

**Comment by ckmackay** [comment]
it is for s-channel, not t-channel: pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/st_sch_4f_ckm_NLO

**Comment by efeyazgan** [comment]
Right! Sorry. I checked s-channel and it is false. So, I assume it is fine.

**Comment by maierbenedikt** [comment]
There is no obvious reason for why this should be fixed. The only scale involved is the top mass, so it can be set to 172.5, but it can also just be left dynamic; shouldn't make much difference.

**Pull request merged**